docs/user/managing-carvel-packages.md

```diff
@@ -72,19 +72,17 @@ This `kapp-controller` plugin is currently being built by default in the Kubeapp
 
 > **TIP**: Please refer to the [getting started documentation](./getting-started.md) for more information on how to install Kubeapps and pass custom configuration values.
 
-In the [values.yaml](../../chart/kubeapps/values.yaml) file, under `kubeappsapis.enabledPlugins` add
-`kapp-controller` to the list of enabled plugins. For example:
+In the [values.yaml](../../chart/kubeapps/values.yaml) file, enable the `packaging.carvel` option (and disable helm, depending on your needs):
```

Comment:
Not sure if we should disable helm in this example, as newcomer users could think both plugins aren't getting along with each other.
I'd suggest just enabling carvel, but no strong opinion though

Reply:
Yeah, I wasn't saying people had to disable helm, just that it may be their intention. But yep, perhaps unnecessary. Removed.